client/benches/tps/lib.rs

```diff
@@ -113,7 +114,9 @@ impl Config {
         let tps = txs_accepted as f64 / elapsed_secs;
         iroha_logger::info!(%tps, %txs_accepted, %elapsed_secs, %txs_rejected);
         if txs_rejected > 0 {
-            return Err(eyre::eyre!("{txs_rejected} transactions were rejected"));
+            // There will be rejected transactions since we submit more than the
+            // network can process.
+            println!("txs_rejected: {}", txs_rejected);
```

Comment:
Debug statement left in. Please convert it to `debug!()`

Reply:
Previously it was an error... Your comment about *more tx then network can process* makes sense. But I still want to know @s8sato 's  opinion

Reply:
I think we can be flexible with this line. Returning error always detects #2078 bug, but on the other hand, it makes the benchmark unmeasurable as long as this bug exists